content/codehimanshu/drone-github-comment/index.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2020-04-24T00:00:00+00:00
+title: Github Comment
+author: codehimanshu
+tags: [ infrastructure, github, drone, comment, buildstatus ]
+logo: github.svg
+repo: gradeup/drone-github-comment
+image: codehimanshu/gitdrone
+---
+
+Use this plugin to update build status on Github Pull Request. This is useful when the complete team does not want to open drone dashboard for each build message.
+
+```yaml
+steps:
+- name: update-status-to-github
+  image: codehimanshu/gitdrone:1.7
+  pull: true
+  environment:
```

Comment:
This plugin seems to not follow plugin best practices. If your env variables would accept a PLUGIN_ prefix for the env variables you could simply use the settings map.

Reply:
The plugin is not using any PLUGIN_ environment variables, is it `drone_access_token` which needs to be changed instead of picking from secrets?

Reply:
In order to register a plugin with the plugin index, the plugin should follow best practices and use settings to configure the plugin, as opposed to environment variables.